src/checkers/inference/model/serialization/ToStringSerializer.java

```diff
@@ -182,16 +182,19 @@ public String serialize(ConstantSlot slot) {
         stringBuilder.append("( ");
         stringBuilder.append(slot.getId());
         stringBuilder.append(" => ");
-        String fullAnno = slot.getValue().toString();
-
+        String fullAnno = slot.getValue().toString().substring(1);
+        DeclaredType annoType = slot.getValue().getAnnotationType();
```

Comment:
The annoType local variable is only used in the "then" branch below. Can you move the variable there?